app/Nova/Team.php

```diff
@@ -159,6 +161,39 @@ protected function controlFields(): array
         ];
     }
 
+    /**
+     * Remote attendance fields.
+     *
+     * @return array<\Laravel\Nova\Fields\Field>
+     */
+    protected function remoteAttendanceFields(): array
+    {
+        return [
+            Text::make('Link', 'attendance_secret')
+                ->hideFromIndex()
+                ->resolveUsing(static function (?string $secret): ?string {
+                    return null === $secret ? null : config('app.url').'attendance/remote/'.$secret;
+                })
+                ->readonly(static function (Request $request): bool {
+                    return ! $request->user()->hasRole('admin');
+                })
```

Comment:
I don't think this should be editable at all. If it is going to be editable, there should be a separate Nova field for the secret itself with no custom resolve function so you don't end up with the attendance_secret having the `https://apiary-test.robojackets.org/attendance/remote/` prefix in it (which makes the link resolve as `https://apiary-test.robojackets.org/attendance/remote/https://apiary-test.robojackets.org/attendance/remote/randomcharactershere` if you don't change it).

Reply:
In the interest of keeping admin superpowers 🙂 I'd like to keep it editable, though I will make a separate field as you suggest.